templates/focus-ios.xliff

```diff
@@ -306,6 +333,10 @@
         <source>Send usage data</source>
         <note>Label for Send Usage Data toggle on main screen</note>
       </trans-unit>
+      <trans-unit id="Settings.whatsNewTitle">
+        <source>What's New</source>
```

Comment:
We should be using "proper" apostrophes in the strings: ’ instead of '
I believe it's ok in the notes to use whichever, but not under source

Reply:
Confirmed: all user facing strings should use an apostrophe.